.github/workflows/geolocator_apple.yaml

```
@@ -180,7 +180,7 @@ jobs:
     runs-on: macos-latest
 
     env:
-      xcodebuild-destination: "platform=iOS Simulator,OS=15.2,name=iPhone 13"
+      xcodebuild-destination: "platform=iOS Simulator,OS=16,name=iPhone 14"
```

```suggestion
      xcodebuild-destination: "platform=iOS Simulator,OS=16.0,name=iPhone 14"
```
This should fix the IOS Test